internal/testhelpers/e2e/shell.go

```diff
@@ -0,0 +1,31 @@
+package e2e
+
+import (
+	"strings"
+
+	"github.com/ActiveState/cli/internal/osutils"
+)
+
+type Shell string
+
+const (
+	Bash Shell = "bash"
+	Zsh        = "zsh"
+	Tcsh       = "tcsh"
+	Fish       = "fish"
+	Cmd        = "cmd.exe"
```

Comment:
Might be good if we hook this up to the constants in subshell, eg. https://github.com/ActiveState/cli/blob/5dc25c28a3dd85bddbfbc22a8fd68f5205025e1b/internal/subshell/bash/bash.go#L42-L42

Reply:
These are actually executable names, not shell names (although they are mostly equivalent). These strings are passed to Spawn(), so they need to exist on $PATH. There's no guarantee that a given shell's `Name` string will exist on $PATH.